pkg/plugin/plugin.go

```diff
@@ -183,6 +185,45 @@ func Uninstall(name string) error {
 	return os.RemoveAll(pluginDir)
 }
 
+// Information gets the information about an installed plugin
+func Information(name string) (string, error) {
+	pluginDir := filepath.Join(dir(), name)
+
+	if _, err := os.Stat(pluginDir); err != nil {
+		if os.IsNotExist(err) {
+			return fmt.Sprintf("Could not find a plugin called '%s', did you install it?", name), nil
```

Comment:
Can we return an error instead?

```suggestion
			return "", xerrors.Errorf("Could not find a plugin called '%s', did you install it?", name)
```

The output will be clear.

```
$ trivy plugin info foo
2021-12-12T11:33:14.040+0900    FATAL   plugin information display error: Could not find a plugin called 'foo', did you install it?
```